openshift-gitops/infra/base/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: openshift-gitops
+resources:
+  - application.yaml
+  - projects.yaml
```

Comment:
The openshift-gitops/infra is for deploying the s/w infra for gitops. It will have configuration for installing the gitops operator, installing, initialization and unsealing of the vault, setting up of project that will be used by the apps. The application.yaml will move to openshift-gitops/app.yaml location as previously.

Reply:
The app.yaml will turn in future as pattern1...n.

Reply:
ok, moving it outside of infra